crates/valence_generated/build/item.rs

```diff
@@ -193,6 +196,27 @@ pub(crate) fn build() -> anyhow::Result<TokenStream> {
             pub snack: bool,
         }
 
+        impl Serialize for ItemKind {
```

Comment:
The imports from serde should use the fully qualified name, not imported with `use` (macro hygiene)